camera_system/camera_calibrator.py

```diff
@@ -0,0 +1,107 @@
+"""カメラキャリブレーションモジュール.
+
+カメラキャリブレーションを行う
+@author kawanoichi
+"""
+import cv2
+from typing import List
+from color_changer import Color, ColorChanger
+from camera_coordinate_calibrator import CameraCoordinateCalibrator
+from game_info import GameInfo
+
+
+class CameraCalibrator:
+    """ゲームエリア認識クラス.
+
+    Attributes:
+        MODE_AREA_XSIZE: 最頻値を求める範囲のxサイズ(奇数)
+        MODE_AREA_YSIZE: 最頻値を求める範囲のyサイズ(奇数)
+    """
+
+    # テンプレートサイズ(奇数)
+    MODE_AREA_XSIZE = 5
+    MODE_AREA_YSIZE = 5
+
+    def __init__(self, read_path: str):
+        """CameraCalibrationのコンストラクタ.
+
+        Args:
+            read_path: コース画像パス
+        """
+        self.__calibration_img = cv2.imread(read_path)
```

Comment:
やっぱり，これをCameraInterfaceで直接取得するようにした方がいいと思う
同じシステム内なのにファイルを介してやり取りするっておかしくない？

だからCameraInterface.capture_frame()の戻り値に`cv2.imread(frame)`にして，ここで呼び出すべきだと思う
```suggestion
        self.__calibration_img = camera_interface.capture_frame()
```